mmgen/core/hooks/ema_hook.py

```diff
@@ -27,14 +27,17 @@ class ExponentialMovingAverageHook(Hook):
             Defaults to 'lerp'.
         interp_cfg (dict | None, optional): Set arguments of the interpolation
             function. Defaults to None.
-        interval (int, optional): Evaluation interval (by epochs). Default: -1.
+        interval (int, optional): Evaluation interval (by iterations).
+            Default: -1.
+        start_iter (int, optional): Start iteration for ema. Default: 1.
```

Comment:
Please specify the behavior of `start_iter` so that users can know its functionality.